weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -192,9 +139,9 @@ def _flush_calls(
             self._flush_calls(batch[split_idx:], _should_update_batch_size=False)
             return
 
-        # If a single item is too large, we can't send it - raise an error
+        # If a single item is too large, we can't send it -- log an error and drop it
         if encoded_bytes > self.remote_request_bytes_limit and len(batch) == 1:
-            raise ValueError(
+            logger.error(
```

Comment:
I think we missed something here. We don't actually drop the call, right? We should like return or something... cc @andrewtruong

Reply:
https://github.com/wandb/weave/pull/4191